server/api/users.js

```diff
@@ -1,10 +1,53 @@
 const express = require('express')
 const router = express.Router()
-const { User, Session, Cart, CartItem, Order, OrderItem } = require('../db')
+const { User, Session, Cart, CartItem } = require('../db')
 const bcrypt = require('bcrypt');
+const { response } = require('express');
 
 const A_WEEK_IN_SECONDS = 60 * 60 * 24 * 7;
 
+// GET /api/users
+router.get('/', async(req, res, next) => {
+  try {
+      if(req.user.isAdmin) {
+        console.log('\nAuthorized...\n')
+        const users = await User.findAll()
+        res.send(users)
+      } 
+  } catch (error) {
+      console.log('\nGo Away...\n')
+      res.status(401)
+  }
+})
+
+// DELETE /api/users/:userID
+router.delete('/:userID', async(req, res, next) => {
+  try {
+    if(req.user.isAdmin) {
+      await User.destroy({where: {id: req.params.userID, isAdmin: false}})
+      res.status(200)
+    } 
+  }
+  catch (error){
+    console.log('\nGo Away...\n')
+    res.status(401)
+    next(error)
+  }
+});
+
+router.put('/:userID', async(req, res, next) => {
```

Comment:
Should the put route be protected as well? I am not sure. A user needs to be able to 'put' their reset password in the DB, but only an admin can put a change to the isAdmin status, so it's more complicated than other routes. Maybe should be two separate routes, one for pw reset, and this one can just be admin protected like the others here.

Reply:
You're right. I had it protected but must've done something careless to take it out. Also, yes, I think a separate route for pw reset would make sense.